lib/query/querycompiler.js

```diff
@@ -199,10 +199,21 @@ class QueryCompiler {
     );
   }
 
+  _optimizerHints() {
+    let hints = this.grouped.optimizerHints || [];
+    hints = hints.map((hint) => compact(hint.value).join(' '));
+    hints = compact(hints).join(' ');
+    if (hints) {
+      return `/*+ ${hints} */ `
```

Comment:
Later we may add an optional parameter to configure the `+` mark in the hint if there are some dialects or proxy / server interprets that require different marks. I don't know about any so far though.